README.md

```diff
@@ -225,3 +225,37 @@ Create and publish NPM package and WebJar
+
+* frontendCommitToCheckout: ladybug-frontend commit to checkout, typically a reference to a tag.
+* backendCommitToCheckout: ibis-ladybug commit to checkout, typically a branch name.
+* useRealFrontend: `true`, because we do not want to use the checkout of ladybug-frontend, but the WebJar contained in ibis-ladybug.
+* mergeMasterToBranch: `true`, for the backend directs the test to merge the master into the checkout. This way, a pull request of the backend is emulated.
+
+This scenario does not work on forks of the ibis-ladybug project. A fork does not have rights to trigger tests within the `ibissource/ladybug-frontend` project. A commit on `ibissource/ibis-ladybug` triggers the front-end test if the ibis-ladybug `pom.xml` or the ibis-ladybug test script is updated. GitHub will not show the success or failure of the front-end test. Please browse to https://github.com/ibissource/ladybug-frontend and go to "Actions". Check the latest test run there.
+
+The second use of the Cypress test is testing the front-end. The Cypress test is triggered by every commit and every pull request update of the ladybug-frontend project. The input arguments are not set, causing the new commit (or merge commit for a PR) to be checked out for ladybug-frontend. For ibis-ladybug the master is checked out. Input `useRealFrontend` is undefined, which is equivalent to `false`. This means that the front-end comes from the checkout of ladybug-frontend; the WebJar inside ibis-ladybug is ignored. Finally, `mergeMasterToBranch` is undefined (`false`). This scenario also works on forks of `ladybug-frontend`.
+
+Here is how to use the CI/CD of Ladybug:
+
+### Update the backend only
+
+If the backend is updated without front-end changes, no GitHub actions are triggered. Rely on the Jenkins test of ibis-ladybug to test your change.
+
+### Update the front-end
+
+When updating the front-end, work on a branch that is allowed to live on a fork of the ladybug-frontend project. The Cypress test is executed for every commit. If you turn your work into a pull request, the success or failure status of the Cypress test will be shown in the pull request. As long as the updated front-end code is not referenced by the ibis-ladybug project, it is not part of Ladybug.
+
+### Update front-end and backend
+
+Update the front-end as explained above. Release the front-end in a WebJar as explained in [Create and publish NPM package and WebJar](#create-and-publish-npm-package-and-webjar). Please note that the released commit may live on another branch than master. Ensure that the commit and the tag are pushed to GitHub project ibissource/ladybug-frontend. Then update `pom.xml` of ibis-ladybug to reference the new WebJar; do so on a branch pushed to ibissource/ibis-ladybug. Then browse to https://github.com/ibissource/ladybug-frontend and go to "Actions". Check the latest run of the Cypress test to see whether the change works.
```

Comment:
"Update the front-end as explained above" means using the CICD which will always fail since in this scenario you also changed the backend. It is probably better to mention running the tests locally to check whether it is safe to make a release.

Reply:
Gedaan.